trio_websocket/_impl.py

```diff
@@ -739,7 +738,13 @@ def subprotocol(self):
         '''
         self._str_message += event.data
         if event.message_finished:
-            await self._send_channel.send(self._str_message)
+            try:
+                await self._send_channel.send(self._str_message)
+            except trio.BrokenResourceError:
```

Comment:
Even CPython only makes a "best effort" attempt to keep naive string concatenation cheap – it can totally fail if you get unlucky. Either accumulating in a list or using a `StringIO` should avoid this issue.

Reply:
I've made the improvements separately in #80

Reply:
Thanks, @belm0. I will merge #80 into this branch.